Разметка всех страниц

Schtock opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/36/user/2245591/tasks/5)

:boom: https://htmlacademy-htmlcss.github.io/2245591-sedona-36/1/

Schtock committed:
Добавляет хэдер

Schtock committed:
Добавляет наставника в readme.md

Schtock committed:
Добавляет разметку тела главной страницы

Schtock committed:
Дополняет разметку тела главной страницы

Schtock committed:
Добавляет футер

+ незначительные правки в хэдере и основной части

Schtock committed:
Добавляет страницу каталога

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,148 @@
+            <p class="advantages-description">Седона — не аттракцион для туристов, там течёт своя жизнь</p>
+          </li>
+          <li class="advantages-item-image">
+            <img class="advantages-image" src="" alt="Вывешенные флаги в городке">
```
Это не отдельный пункт списка, это иллюстрация первого пункта.

Schtock replied:
Поправил в новом коммите

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,148 @@
+            <p class="advantages-description">Большинство едет в Гранд Каньон и толпится там</p>
+          </li>
+          <li class="advantages-item-image">
+            <img class="advantages-image" src="" alt="Каньон">
```
Тоже не самодостаточный пункт.

Schtock replied:
Поправил в новом коммите

Dima-Dim commented on index.html:
```diff
@@ -5,8 +5,148 @@
+        <h2 class="visually-hidden">Поиск отелей</h2>
+        <p class="booking-lead">Заинтересовались?</p>
+        <p class="booking-text">Укажите предполагаемые даты поездки,<br>и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+        <a class="button booking-button" href="#">ПОИСК ГОСТИНИЦЫ В седоне</a>
```
Стилизация будем делать с помощью CSS, слова нужно писать так, как мы писали бы их в обычных предложениях.

Schtock replied:
Если речь о переносах строк, то убрал.

Dima-Dim replied:
Речь о заглавных буквах. Вряд ли в обычном предложении напишут "ПОИСК ГОСТИНИЦЫ В седоне".

Schtock replied:
Исправил, вернул переносы строк

Schtock committed:
Вносит корректировки в преимущества

Переносим картинки из отдельных пунктов списка в соседние с текстом.
Убираем переносы строки

Schtock committed:
Убирает заглавный текст в секции booking
